### ISSUE TITLE
Password beginning in a dash fails because BTCRecover thinks it's an option

### ISSUE THREAD

**Issue by aximmaxim** [3rdIteration/btcrecover#307]
btcrecover.py: error: argument --correct-wallet-password: expected one argument

**Comment by 3rdIteration** [comment]
Just enclose it in quotes "-passwordwithdash"

**Comment by aximmaxim** [comment]
> Just enclose it in quotes "-passwordwithdash"

Didn't work, same error.

**Comment by 3rdIteration** [comment]
Try single quotes '-passwordwithdash'

**Comment by aximmaxim** [comment]
Same problem. I even cut and pasted '-passwordwithdash' from your message
just in case my keyboard was changing it to smart quotes.

Here's the full command, fwiw

python3 btcrecover.py --wallet MyWallet --dump-privkeys myblock
--correct-wallet-password '-passwordwithdash'

**Comment by 3rdIteration** [comment]
Which operating system are you using?

**Comment by aximmaxim** [comment]
Mac Big Sur 11.4

I have a separate issue with "possible password file" producing recognizable text with multiple iterations of test password. Should I open another issue or post it here?

**Comment by 3rdIteration** [comment]
Just use = as well

python3 btcrecover.py --wallet MyWallet --dump-privkeys myblock
--correct-wallet-password='-passwordwithdash'

For the possible passwords, unless its readable like the examples given in the file then it's not a match.

**Comment by aximmaxim** [comment]
Examples: 

2021-12-30 11:37:21 Possible Password ==>btcr-test-password<== in Decrypted Block ==>{"tx_notes":{},"<==
2021-12-30 11:37:21 Found Password ==>btcr-test-password<== in Decrypted Block ==>{"tx_notes":{},"<==
2021-12-30 11:37:21 Possible Password ==>BTCR-test-passw0rd<== in Decrypted Block ==>{"version":"v2",<==
2021-12-30 11:37:21 Found Password ==>BTCR-test-passw0rd<== in Decrypted Block ==>{"version":"v2",<==
2021-12-30 11:37:21 Possible Password ==>btcr-test-password<== in Decrypted Block ==>[{"type":"HD Key<==
2021-12-30 11:37:21 Found Password ==>btcr-test-password<== in Decrypted Block ==>[{"type":"HD Key<==
2021-12-30 11:37:21 Possible Password ==>btcr-test-password<== in Decrypted Block ==>[{"type":"HD Key<==
2021-12-30 11:37:21 Found Password ==>btcr-test-password<== in Decrypted Block ==>[{"type":"HD Key<==
2021-12-30 11:37:21 Possible Password ==>btcr-test-password<== in Decrypted Block ==>"{\"mnemonic\":\<==
2021-12-30 11:37:21 Found Password ==>btcr-test-password<== in Decrypted Block ==>"{\"mnemonic\":\<==

**Comment by 3rdIteration** [comment]
well yea thats from runnings the tests, it has nothing to do with your wallet

**Comment by aximmaxim** [comment]
ahhh ok! That makes more sense! I didn't realize I'd run the tests in middle of a brute-force run. Thanks for all your help.

**Comment by aximmaxim** [comment]
I don't think this is an issue with BTCRecover, so I'm loathe to open an
issue, but I can't figure out the problem or its solution. I've been
running BTCrecover for a couple of weeks but I hadn't found the password.
Nothing has changed in system, afaik, from the last time it ran
successfully. But yesterday I had a system crash and when I tried to run it
again, I get the following message:

btcrecover-master % python3 btcrecover.py --restore savefile.txt

Traceback (most recent call last):

  File "/Users/username/Downloads/btcrecover-master/btcrecover.py", line
30, in <module>

    from btcrecover import btcrpass

  File
"/Users/username/Downloads/btcrecover-master/btcrecover/__init__.py", line
2, in <module>

    from .btcrpass import *

  File
"/Users/username/Downloads/btcrecover-master/btcrecover/btcrpass.py", line
46, in <module>

    from Crypto.Cipher import AES

ModuleNotFoundError: No module named 'Crypto'


I tried to uninstall and reinstall crypto modules and that did nothing. I
tried to uninstall crypto and install and pycroptodome3 but it says No
matching distribution found for pycroptodome3.


Any advice appreciated!
